src/sentry.ts

```diff
@@ -66,10 +65,12 @@ provider.register({
     contextManager: new Sentry.SentryContextManager(),
 });
 
+
 registerInstrumentations({
-    instrumentations: [new ExpressInstrumentation(), new HttpInstrumentation()],
```

Comment:
Express has to be added if you care about it, only http can/should be skipped here.

Reply:
Gotcha - that makes sense. I thought this could be it. But since `Sentry.httpIntegration` took care of OTEL httpInstrumentation, i thought the Sentry Express integration may do the same.